ignite/services/chain/chain.go

```diff
@@ -186,7 +186,12 @@ func (c *Chain) RPCPublicAddress() (string, error) {
 		if err != nil {
 			return "", err
 		}
-		validator := conf.Validators[0]
+
+		validator, err := config.FirstValidator(conf)
```

Comment:
This is actually a part where we wouldn't want to require a validator but return a default value if no validator is defined

Reply:
Fixed in 59fd322